api/config/cable.yml

```diff
@@ -2,9 +2,9 @@ development:
   adapter: async
 
 test:
-  adapter: async
+  adapter: test
 
 production:
   adapter: redis
-  url: redis://localhost:6379/1
+  url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
```

Comment:
What's with the hard-coded URL and port?

Reply:
This was autogenerated. We don't need ActionCable for our use cases. I will remove it in a future commit.